hydra-node/test/Hydra/Chain/Direct/TxSpec.hs

```diff
@@ -194,6 +196,21 @@ spec =
                         & counterexample "Failed to construct and observe init tx."
                         & counterexample (renderTx tx)
 
+      prop "Ignore InitTx with wrong contestation period" $
+        withMaxSuccess 60 $ \txIn cperiod (party :| parties) cardanoKeys -> do
+          i <- arbitrary `suchThat` (> 0)
+          let params = HeadParameters cperiod (party : parties)
+              (UnsafeContestationPeriod cp) = cperiod
+              -- construct different/wrong CP
+              wrongCPeriod = UnsafeContestationPeriod $ cp + wordToNatural i
```

Comment:
Should: just generate a `i :: Natural`?

Reply:
We have overlapping instances for Arbitrary Natural in scope here and it is not so easy to resolve this error so I didn't want to waste any time.